cmake/Platform/Android/Configurations_android.cmake

```diff
@@ -56,7 +56,6 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
 
             -u ANativeActivity_onCreate
 
-            -L${LY_NDK_ABI_ROOT}/usr/lib
```

Comment:
So the NDK on Windows has this set, but not NDK on Linux?

Reply:
I guess a regular Windows system does not have /usr/lib containing conflicting files

Reply:
On Windows this line should have no effect at all as it likely sets a non-existent library path, but it breaks the build on linux and probably macos as well

Reply:
https://github.com/o3de/o3de/search?q=LY_NDK_ABI_ROOT this is the single occurrence of LY_NDK_ABI_ROOT so the removed line should expand to `-L/usr/lib` on Windows as well and not to a path within NDK

Reply:
It also appears that `LY_NDK_SRC_ROOT` is also never set

Reply:
sure!